setup.py

```diff
@@ -26,10 +26,10 @@
     install_requires=(
         "click>=7",
         # Pin these to avoid unplanned messy diffs.
-        "black==19.10b0",
+        "black==20.8b1",
```

Comment:
Could these all use tilde range like `isort` to just lock `major.minor`?

Reply:
uhm, maybe?